chart/docker-auth/templates/configmap.yaml

```diff
@@ -9,8 +9,16 @@ data:
     token:
       issuer: "{{ .Values.configmap.data.token.issuer }}"  # Must match issuer in the Registry config.
       expiration: {{ .Values.configmap.data.token.expiration }}
+{{- if .Values.secret.certificateName }}
+      certificate: "/config/certs/{{ .Values.secret.certificateName}}"
+{{- else }}
       certificate: "/config/certs/server.pem"
+{{- end }}
+{{- if .Values.secret.keyName }}
```

Comment:
Since `keyName` will have a default, check for `.Values.secret.secretName` instead.

Reply:
You can also merge this if statement with the one above because they will be checking for the same thing.